Parsers/ASimFileEvent/test/SentinelOne_vimFileEvent_SchemaTest.csv

```diff
@@ -0,0 +1,87 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [SrcFilePathType]"
+"(1) Warning: Missing recommended field [SrcIpAddr]"
+"(2) Info: Missing optional alias [Application] aliasing non-existent column [TargetAppName]"
+"(2) Info: Missing optional alias [Url] aliasing non-existent column [TargetUrl]"
+"(2) Info: Missing optional field [ActingAppId]"
+"(2) Info: Missing optional field [ActingAppName]"
+"(2) Info: Missing optional field [ActingAppType]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUpn]"
+"(2) Info: Missing optional field [ActorUserAadId]"
+"(2) Info: Missing optional field [ActorUserPuid]"
+"(2) Info: Missing optional field [ActorUserSid]"
+"(2) Info: Missing optional field [ActorUserType]"
```

Comment:
can we set this?
https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/lib/functions/ASIM_GetUserType.yaml

Reply:
Done